src/main/scala/xiangshan/backend/fu/NewCSR/CSREvents/TrapEntryHSEvent.scala

```diff
@@ -116,7 +116,7 @@ class TrapEntryHSEventModule(implicit val p: Parameters) extends Module with CSR
   out.stval    .valid := valid
   out.htval    .valid := valid
   out.htinst   .valid := valid
-  out.targetPc .valid := valid
+  out.targetPc .valid := GatedValidRegNext(valid)
```

Comment:
Please convert in.pcFromXtvec to ValidIO(PC), and do RegNext for valid at the same place as the exception pc